build-script.py

```diff
@@ -861,6 +813,31 @@ def add_default_build_arguments(parser: argparse.ArgumentParser) -> None:
             "-v", "--verbose", action="store_true", help="Enable verbose logging."
         )
 
+    def add_generate_source_code_args(parser: argparse.ArgumentParser) -> None:
+            parser.add_argument(
+                "--toolchain",
+                required=True,
+                help="The path to the toolchain that shall be used to build SwiftSyntax.",
+            )
+
+            parser.add_argument(
+                "--gyb-exec",
+                default=GYB_EXEC,
+                help="Path to the gyb tool (default: %(default)s).",
+            )
+
+            parser.add_argument(
+                "--gyb-only",
+                action="store_true",
+                help="""
+                Only generate gyb templates (and not generate-swift-syntax-builder's templates)
```

Comment:
I know this wasn't a change introduced in this PR, but I think Python doesn't automatically dedent multi-line strings like Swift, so we should probably use something like `.strip()` or `textwrap.dedent` to avoid ending up with strange formatting in the help output.

Reply:
Good idea. I’ll set up another PR for it.

Reply:
I just checked and these messages get automatically demented because we use `argparse.RawDescriptionHelpFormatter`.

Reply:
Ah interesting, didn't know about that, nice! 👍